profiles/cclcc/sprites.lua

```diff
@@ -78,7 +78,33 @@ root.SpriteSheets = {
         Path = { Mount = "system", Id = 18 },
         DesignWidth = 2048,
         DesignHeight = 3072
-    }
+    },
+    ["YNTRG_BG0"] = {
```

Comment:
I'd prefer if we followed the naming style of all the other sprites to keep consistency.

Reply:
> I'd prefer if we followed the naming style of all the other sprites to keep consistency.

Yeah I just used the file names for those